apps/api/views.py

```diff
@@ -0,0 +1,45 @@
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+from rest_framework import serializers
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
+
+from apps.api.permissions import HasUserAPIKey
+from apps.experiments.models import Experiment, Participant, ParticipantData
+
+
+class ExperimentSerializer(serializers.Serializer):
+    name = serializers.CharField()
+    experiment_id = serializers.UUIDField(source="public_id")
+
+
+@api_view(["GET"])
+@permission_classes([HasUserAPIKey])
+def get_experiments(request):
+    data = []
+    for experiment in Experiment.objects.filter(team__slug=request.team.slug).all():
+        data.append(ExperimentSerializer(experiment).data)
+    return Response(data=data)
```

Comment:
I think we should generally use pagination even in cases where it may not seem necessary so that we get consistent responses: https://www.django-rest-framework.org/api-guide/pagination/#pagination

If you still want to use function based views you can do this: https://stackoverflow.com/a/34043578

I would also vote for using cursor based pagination though I could be persuaded: https://www.django-rest-framework.org/api-guide/pagination/#cursorpagination